tools/internal_ci/linux/grpc_xds_bazel_python_test_in_docker.sh

```diff
@@ -64,7 +64,7 @@ bazel build //src/python/grpcio_tests/tests_py3_only/interop:xds_interop_client
 # because not all interop clients in all languages support these new tests.
 GRPC_VERBOSITY=debug GRPC_TRACE=xds_client,xds_resolver,xds_cluster_manager_lb,cds_lb,xds_cluster_resolver_lb,priority_lb,xds_cluster_impl_lb,weighted_target_lb "$PYTHON" \
   tools/run_tests/run_xds_tests.py \
-    --test_case="all,path_matching,header_matching" \
+    --test_case="all,path_matching,header_matching,circuit_breaking" \
     --project_id=grpc-testing \
     --source_image=projects/grpc-testing/global/images/xds-test-server-2 \
```

Comment:
Just noticed this: does this need to be updated to `xds-test-server-3`? cc @menghanl

Reply:
I was not aware of this update. I've seen the test pass manually, but CI hasn't run the test on master yet. I'll wait on @menghanl 's response before merging.

Reply:
Yes, needs to be server 3. We need the new server to block the RPCs, so they stay in-progress.

Reply:
Was the test passing on master without this change?
cc @voidzcy

Reply:
Oh! Good catch! It has to use `xds-test-server-3`. I am highly doubt the test would pass by using `xds-test-server-2`, we had run into a debug hell by using `xds-test-server-2`.

Reply:
Oh yeah, the test is failing on the dashboard.

Reply:
@stanley-cheung Nice catch! [Fixing on master first](https://github.com/grpc/grpc/pull/25166).

Reply:
Merged the fix on master. Just going to wait for a green build on master before going forward with this PR.

Reply:
There's been a [green run on master](https://fusion2.corp.google.com/invocations;query=grpc_xds_python/574a45b8-7c74-4f46-8d33-0bc3fc7bb848/artifacts/github%2Fgrpc%2Freports%2Fcircuit_breaking;config=default). This should be good to go now.

Reply:
LGTM. Thanks everyone!